### PR TITLE
Show window sooner

### PR DESCRIPTION
**Pull request by kevinsawicki**
This changes Atom to show the editor window as quickly as possibly, before package and theme loading begins.

It stores the background color in local storage to minimize the color changes once the editor loads.

### CODE REVIEW

**Comment by kevinsawicki** [comment]
@benogle want to give this branch a try when you get a chance?

**Comment by benogle** [comment]
Yeah will do in a few

File: src/atom.coffee [hunk 1]
```diff
@@ -572,6 +581,13 @@ class Atom extends Model
     dimensions = @getWindowDimensions()
     @state.windowDimensions = dimensions if @isValidDimensions(dimensions)
 
+  storeWindowBackground: ->
```
Comment (mnquintana) [inline src/atom.coffee]: I'm actually really excited about this :raised_hands:

Comment (mnquintana) [inline src/atom.coffee]: Would this also resolve https://github.com/atom/atom/issues/929? (Seems like it would.)

**Comment by Zireael07** [comment]
Great!

File: src/atom.coffee (hunk 1 above)
Comment (kevinsawicki) [inline src/atom.coffee]: @mnquintana I'm not sure it will, I think that was an underlying Chrome issue that is still present.

**Comment by kevinsawicki** [comment]
shipping this :fountain:

**Comment by benogle** [comment]
:tractor:

**Comment by mark-hahn** [comment]
What does the tractor mean?  I always feel like I'm left out of an inside
joke.

**Comment by benogle** [comment]
He's plowing, man

**Comment by kevinsawicki** [comment]
> What does the tractor mean? 

My interpretation of it is sometimes "shipping" with a :ship: gets old, so people sprinkle in other vehicle types like :speedboat: :dash:, :boat:, :car: :dash:, and :tractor: just to add a little variety to signal that something is "moving", "motoring", "sailing", etc. along.

**Comment by yongkangchen** [comment]
+1

**Pull request merged**